src/code/z_actor.c

```diff
@@ -1084,45 +1084,45 @@ s32 func_8002E020(Actor* actorA, Actor* actorB, s16 arg2) {
     return false;
 }
 
-s32 func_8002E084(Actor* actor, s16 arg1) {
-    s16 var = actor->yawTowardsPlayer - actor->shape.rot.y;
+s32 Actor_YawInRangeWithPlayer(Actor* actor, s16 yaw) {
+    s16 yawDiff = actor->yawTowardsPlayer - actor->shape.rot.y;
 
-    if (ABS(var) < arg1) {
+    if (ABS(yawDiff) < yaw) {
         return true;
     }
 
     return false;
 }
 
-s32 func_8002E0D0(Actor* actorA, Actor* actorB, s16 arg2) {
-    s16 var = Actor_WorldYawTowardActor(actorA, actorB) - actorA->shape.rot.y;
+s32 Actor_YawInRangeWithActor(Actor* actorA, Actor* actorB, s16 yaw) {
```

Comment:
I'm not sure about the name for these functions because the "InRange" part makes it sound like it's doing some kind of distance check, but I also can't think of anything better, aside from maybe reordering terms to make it clear it's doing a range check with the yaw, not between the actors or anything like that.

Reply:
do you have a specific example of how to reorder the terms? I kinda think this is clear, since i tried to make it sound like the sentence "the yaw is in range with the actor"
vs the other variation of the function which says "yaw and distance is in range"

but if its not clear to others then we can def think of something else

Reply:
just bumping this and the comments above. still not sure on the best route to take

Reply:
pls 🙏

Reply:
I don't know what the best route is either, but I think the current names could be confusing and the sentence "the yaw is in range with the actor" doesn't really make sense to me.
To me it looks like these functions are checking that the yaw difference between 2 actors is smaller than a certain value, so maybe we could just call them "Actor_YawCheck" (or "Actor_ActorYawCheck") and "Actor_PlayerYawCheck", with a similar pattern for the yaw + distance checks.
Also we should probably consider func_8002E020 and func_8002E084 while naming these, since they are doing similar types of checks but from the other actor's perspective (?)

Reply:
to me the "less than a certain value" is the range check. its looking for the angle difference to be within a certain range.
Since its not looking for an exact value for the difference, but if the actors are facing each other to a certain extent, the check will pass

Reply:
I agree that theres probably something better. but something like yaw check is too ambiguous to me

Reply:
My main concern is the term "range" makes it sound like it's doing a distance check to me, especially when saying "in range with the actor/player". I think a different order like "Actor_YawWithActorInRange" would already be less confusing, but it's not ideal either.

Reply:
yea dunno, range doesnt mean just distance to me. maybe theres a totally different direction to take this thats better, if anyone has ideas

Reply:
Perhaps include the word `Arc`? `YawInArcRange` or similar?

Reply:
Possibly? I feel like i would just be more confused when reading the name for the first time, trying to figure out how an arc plays into it. I understand what youre going for though 

"yaw in range" is sufficient to me, because there is a range of angle values that will satisfy the check. range doesnt mean just distance, to me. And when distance is involved, the name specifies that its checking for both the yaw and the distance to be within a specified range.

we use the phrasing "yaw toward player" for that one actor struct variable, so maybe using that kind of phrasing would help? not sure